lms/services/assignment.py

```diff
@@ -214,32 +217,51 @@ def is_member(self, assignment: Assignment, h_userid: str) -> bool:
     def get_assignments(
         self,
         instructor_h_userid: str | None = None,
+        admin_organization_ids: list[int] | None = None,
```

Comment:
Same pattern as for courses, fetch assignments where the users is an instructor **or** an admin.